code/modules/mining/ores_coins.dm

```diff
@@ -7,6 +7,11 @@
 	var/points = 0 //How many points this ore gets you from the ore redemption machine
 	var/refined_type = null //What this ore defaults to being refined into
 
+/obj/item/weapon/ore/ex_act(severity)
+	if(refined_type)
+		new refined_type(get_turf(src.loc))

```

Comment:
Actually I just realized I needed to do it anyways, as sheets aren't as easily transported as ores.